### PR TITLE
Fix #769 - Add custom reminder notification names

### LINKED ISSUE #769: Add custom reminder name to notification
**Summary:** 

Currently, you can set a custom reminder name so it shows up in the reminders list, but that name is not shown in the actual reminder notification - we currently just show "Route 5 is arriving in 7 min!".

It would be helpful to add the reminder name to the notification itself so riders can more easily differentiate them.

The reminder notifications could use a UI update in general, too.

**Steps to reproduce:** 

1. Tap on an arrival time
1. Choose "Set a reminder"
1. Set a reminder (if you want to be immediately notified, set it for more than the current ETA)
1. Look at the reminder notification

**Expected behavior:** 

The notification should include the reminder name

**Observed behavior:** 

The notification does not include the reminder name - it just says "Route 5 is arriving in 7 min!".

**Device and Android version:** 

N/A

### PR DESCRIPTION
**Pull request by bridgette**
#769 Replace app name in notification title with user-inputted notification name. Small style fixes to changed files.

__________________

**Existing**

Existing reminder notification; title is app name.

existing

_______________________________________________

**Proposed**

User sets reminder name as before... 
proposed-1

... which is the new notification title.
proposed-2

___________________________________________

- [X] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [X] Run the unit tests with `gradlew connectedObaGoogleDebugAndroidTest connectedObaAmazonDebugAndroidTest` to make sure you didn't break anything

- [X] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: onebusaway-android/src/main/java/org/onebusaway/android/tripservice/PollerTask.java [hunk 1]
```diff
@@ -151,7 +156,6 @@ private long getReminderMin(String tripId, String stopId) {
      * Checks arrivals from given ObaArrivalInfoResponse
      *
      * @param response arrival information
-     * @param tripId
```
Comment (barbeau) [inline onebusaway-android/src/main/java/org/onebusaway/android/tripservice/PollerTask.java]: Looks like the Javadoc for `tripId` was removed - could you please re-add with short description?

Comment (bridgette) [inline onebusaway-android/src/main/java/org/onebusaway/android/tripservice/PollerTask.java]: not sure how `tripId` got deleted-- might have been the code formatter. I'll add it back in with a description 👍

**Comment by barbeau** [comment]
Thanks @bridgette!  Do you also want to add your name to the end of the contributors list, both in `CONTRIBUTORS.md` and in strings.xml `about_text`?

**Comment by barbeau** [comment]
Thanks @bridgette, this is great!

If you're interested, there is another reminder-related enhancement at https://github.com/OneBusAway/onebusaway-android/issues/609 you might want to take a look at.  Or whatever else piques your interest ([your first pr](https://github.com/OneBusAway/onebusaway-android/labels/your%20first%20PR) issues are a good place to start) - I appreciate the contributions!

**Pull request merged**